ui/src/theme.js

```diff
@@ -35,7 +36,25 @@ export default createTheme({
     },
   },
   typography: {
```

Comment:
Can we update the them to use the protocol pallet?

https://github.com/mozilla-releng/shipit/pull/1339/files#diff-f0f2b454dd5fb88af32df6fdc3f078409c63663d7251463a2a3c7304ef56be81R35-R46

https://protocol.mozilla.org/docs/fundamentals/color.html